pkg/ebpf/netlink/conntracker_test.go

```diff
@@ -152,19 +152,24 @@ func TestGetUpdatesGen(t *testing.T) {
 
 	rt.register(c)
 	var last uint8
+
+	// there will be two entries in rt.state for the entry we just registered.
+	// set them both to be 5 generations older than they are
+	require.Len(t, rt.state, 2)
 	for _, v := range rt.state {
 		v.expGeneration -= 5
 		last = v.expGeneration
-		break // there is only one item in the map
 	}
 
 	iptr := rt.GetTranslationForConn(util.AddressFromString("10.0.0.0"), 12345, process.ConnectionType_tcp)
 	require.NotNil(t, iptr)
 
-	for _, v := range rt.state {
-		assert.NotEqual(t, last, v.expGeneration)
-		break // there is only one item in the map
-	}
+	require.Len(t, rt.state, 2)
+	require.Len(t, rt.shortLivedBuffer, 0)
+	entry := rt.state[connKey{
+		util.AddressFromString("10.0.0.0"), 12345, process.ConnectionType_tcp,
+	}]
+	assert.NotEqual(t, entry.expGeneration, last)
```

Comment:
nit, since `NotEqual` doesn't print the expected value, can you add it as a message ?

Reply:
Good call, done